openfl-tutorials/interactive_api/PyTorch_MedMNIST_2D/requirements.txt

```diff
@@ -0,0 +1,3 @@
+medmnist
+fire==0.4.0
```

Comment:
I'm not sure if fire and PIL are needed. I just had to install medmnist and I could run the tutorial.
Also, let's put this requirements file under envoy/.